assets/perplexity.yaml

```diff
@@ -49,7 +49,10 @@
   organization: Perplexity
   description: Perplexity Ask is a new search interface that uses advanced artificial
     intelligence technologies
-  created_date: unknown
+  created_date:
+    value: 2022-12-07
+    explanation: Perplexity Ask was announced in a [[Twitter post]](https://twitter.com/perplexity_ai/status/1600551871554338816?s=20)
+      by the company.
   url: https://www.perplexity.ai/
   dependencies: [Perplexity model]
```

Comment:
The [announcement post](https://twitter.com/perplexity_ai/status/1600551871554338816) said "Announcing Perplexity Ask, a new search interface that uses OpenAI GPT 3.5 and Microsoft Bing to directly answer any question you ask."

Should dependencies be changed to [GPT 3.5, Bing Search] ?

Reply:
yes i think so

Reply:
i.e. i think you can remove the dataset + model nodes for perplexity and just have application with these dependencies.